### DISCUSSION TITLE
Adding a Training rank for maps

### DISCUSSION THREAD

**Discussion by Baldahegao** [ppy/osu#15017] [Ideas]
Training maps should have their own separate Rank, as they are currently all graveyarded, which makes it hard to differentiate for new players which maps are useful/good for training.

I don't think a leaderboard is needed for them, so (hopefully) it shouldn't be too hard to implement into the game.

**Comment by smoogipoo** [comment]
While I can _sort of_ understand this sentiment, new players shouldn't use training maps at all. Literally playing more is the best way to improve until you understand exactly what's wrong with your aim/speed. By that point you'd no longer be a beginner and you'd have some experience with multiplayer/tournaments/high-SR maps.

And for anything more than that, I think it would be better for the community to make collections of maps that fit within a particular skillset on the forums, as there are many maps which could be considered as "training" but also as standard non-training maps.
